src/babel.coffee

```diff
@@ -132,7 +132,7 @@ transpile = (sourceCode, filePath, cachePath) ->
 # either generated on the fly or pulled from cache.
 loadFile = (module, filePath) ->
   sourceCode = fs.readFileSync(filePath, 'utf8')
-  unless sourceCode.startsWith('"use 6to5"') or sourceCode.startsWith("'use 6to5'")
+  unless /^("use 6to5"|'use 6to5'|"use babel"|'use babel')/.test(sourceCode)
```

Comment:
Also, if it matches `6to5` instead of `babel`, I think it should print out a deprecation warning. It's early enough that we can probably get everyone to convert their pragmas now if there's a warning that tells them to.